include/RE/FormComponents/TESForm/TESObjectREFR/Actor/Actor.h

```diff
@@ -533,6 +533,7 @@ namespace RE
 		void						 RemoveExtraArrows3D();
 		bool						 RemoveSpell(SpellItem* a_spell);
 		std::int32_t				 RequestDetectionLevel(Actor* a_target, DETECTION_PRIORITY a_priority = DETECTION_PRIORITY::kNormal);
+		void					  	 ResetAI(UInt32 a_arg1 = 0, UInt32 a_arg2 = 1);
```

Comment:
Sorry, the "UInt32" should be "std::uint32_t", I copy it from the old CommonLibSSE Version and forgot to change the data type.